packages/webvh/src/dids/WebvhDidResolver.ts

```diff
@@ -0,0 +1,128 @@
+import type { AgentContext, DidResolutionResult, DidResolver } from '@credo-ts/core'
+
+import { DidDocument } from '@credo-ts/core'
+import { resolveDID } from 'didwebvh-ts'
+
+import { DIDWebvhCrypto } from './didWebvhUtil'
+
+export class WebvhDidResolver implements DidResolver {
+  public readonly supportedMethods = ['webvh']
+  public readonly allowsCaching = false
+  public readonly allowsLocalDidRecord = false
```

Comment:
Any reason these are both false? For the `did:web` one they are both true, I'd assume the same values here